src/sagemaker_server.cc

```diff
@@ -331,6 +332,16 @@ SagemakerAPIServer::ParseSageMakerRequest(
   }
   (*parse_map)["model_name"] = model_name_string.c_str();
 
+  /* Extract targetModel archive for debugging purposes */
```

Comment:
Should below be removed if it is only for debugging?

Reply:
Actually this is useful for customer to know which model archive is being loaded, I've updated the comment to reflect that, thx